content/en/agent/guide/autodiscovery-management.md

```diff
@@ -25,13 +25,13 @@ Exclude containers from the Agent Autodiscovery perimeter with an exclude rule b
 {{< tabs >}}
 {{% tab "Containerized Agent" %}}
 
-To remove a given Docker container with the image `<IMAGE_NAME>` from Autodiscovery, add the following environment variable to the Datadog Agent:
+In **Agent v7.20+**, to remove a given Docker container with the image `<IMAGE_NAME>` from Autodiscovery, and thus exclude the **logs and metrics**, add the following environment variable to the Datadog Agent:
```

Comment:
should we add a note to say that `AC_INCLUDE` and `AC_EXCLUDE` can be used for prior agent versions? I believe there is a note about its deprecation we can add it there.
Let's see what the docs team say about that as well

Reply:
Yes I agree, let's wait for their reply! Thanks Ahmed!

Reply:
@ahmed-mez yes! let's do that :)

Reply:
here is the original PR https://github.com/DataDog/documentation/pull/7732/files
seems like there are other places we should update